apiary.apib

```diff
@@ -1444,7 +1444,7 @@ Seznam možných událostí:
     <tr><th>event</th><th>popis</th></tr>
     <tr><td>mark_as_sent</td><td>Označit jako odeslanou</td></tr>
     <tr><td>deliver</td><td>Doručit emailem</td></tr>
-    <tr><td>pay</td><td>Označit jako zaplacenou (akceptuje volitelný parametr `paid_at` a `paid_amount`)</td></tr>
+    <tr><td>pay</td><td>Označit jako zaplacenou (akceptuje volitelný parametr `paid_on`, `paid_at`, `paid_amount`, `variable_symbol` a `bank_account_id`)</td></tr>
```

Comment:
We probably should not change the `paid_at` column name, I know it will still work with `paid_at` but it could confuse developers if they are using `paid_at`, and suddenly it is not in the documentation.

Reply:
They are now both in the documentation, I didn't remove anything. Is it ok like that?

Reply:
Ahh, sorry, I didn't see it before, I don't think we should add it to the API v2. It is the same column as `paid_at` so it can be confusing. We should leave it same for the API v2 and do these changes in the upcoming API v3.

Reply:
Ok done, check now.